io.c

```diff
@@ -7754,7 +7755,7 @@ check_pipe_command(VALUE filename_or_command)
  *  <b>File Opened</b>
 
  *  If +path+ does _not_ start with a pipe character (<tt>'|'</tt>),
- *  a file stream is opened with <tt>File.open(path, mode, perm, opts)</tt>.
+ *  a file stream is opened with <tt>File.open(path, mode, perm, open_opts)</tt>.
```

Comment:
`open_opts` should be passed in splatted for keyword arguments.

```suggestion
 *  a file stream is opened with <tt>File.open(path, mode, perm, **open_opts)</tt>.
```

Reply:
Fixed.